core/db_models/strategies/EE_Restriction_Generator_Base.strategy.php

```diff
@@ -54,7 +49,7 @@ abstract class EE_Restriction_Generator_Base
      *
      * @var EE_Default_Where_Conditions[]
      */
-    protected $_cap_restrictions_generated = false;
+    protected $_cap_restrictions_generated;
```

Comment:
Oh, OK. Because otherwise the `has_generated_cap_restrictions()` wouldn't function correctly.